server/controllers/tasks/lib/get_new_suggestions_or_automerge.ts

```diff
@@ -0,0 +1,169 @@
+import CONFIG from 'config'
+import { map, uniq } from 'lodash-es'
+import { getInvEntitiesByClaim } from '#controllers/entities/lib/entities'
+import { getEntitiesByUris } from '#controllers/entities/lib/get_entities_by_uris'
+import getOccurrencesFromEntities from '#controllers/entities/lib/get_occurrences_from_entities'
+import { getOccurrencesFromExternalSources } from '#controllers/entities/lib/get_occurrences_from_external_sources'
+import { haveExactMatch } from '#controllers/entities/lib/labels_match'
+import { prefixifyInv } from '#controllers/entities/lib/prefix'
+import { propertiesValuesConstraints as properties } from '#controllers/entities/lib/properties/properties_values_constraints'
+import { getEntityNormalizedTerms } from '#controllers/entities/lib/terms_normalization'
+import typeSearch from '#controllers/search/lib/type_search'
+import { isNonEmptyString } from '#lib/boolean_validations'
+import { forceArray, someMatch } from '#lib/utils/base'
+import { automerge, validateAndAutomerge } from './automerge.js'
+import { findAuthorWithMatchingIsbnInWikipediaArticles } from './find_authors_with_isbns_in_wikipedia_articles.js'
+
+const { minimumScoreToAutogenerate } = CONFIG.tasks
```

Comment:
[non-blocking]
Does that really need to be configurable? Looks like a hard coded value, that we might want to adjust from time to time, would do just fine

Reply:
Let's roll with that then